app/models/duffel.rb

```diff
@@ -7,6 +7,29 @@ def initialize(contents)
 
   def add_item(item_id)
     contents[item_id.to_s] ||= 0
-    contents[item_id.to_s] += 0
+    contents[item_id.to_s] += 1
+  end
+
+  def total
+    contents.values.sum
+  end
+
+  def count_of(item_id)
+    contents[item_id.to_s]
+  end
+
+# need access, to price, quantity, and total that is inside of item_prices - for each item
```

Comment:
Line is too long. [90/80]<br>Incorrect indentation detected (column 0 instead of 2).